python/ray/worker.py

```diff
@@ -2125,12 +2130,22 @@ def cancel(object_ref: ray.ObjectRef, *, force: bool = False, recursive: bool =
     worker = ray.worker.global_worker
```

Comment:
Can we add a section on what user should expect when it returns/raises in the comment? Thanks!